### PR TITLE
Add helper: ride_has_ratings

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

File: src/openrct2/peep/peep.c [hunk 1]
```diff
@@ -10597,25 +10597,22 @@ static sint16 peep_calculate_ride_value_satisfaction(rct_peep* peep, rct_ride* r
...
 		return 70;
 	}
-	intensitySatisfaction = 3;
-	nauseaSatisfaction = 3;
 
-    maxIntensity = (peep->intensity >> 4) * 100;
-    minIntensity = (peep->intensity & 0xF) * 100;
+	uint8 intensitySatisfaction = 3;
+	uint8 nauseaSatisfaction = 3;
+    ride_rating maxIntensity = (peep->intensity >> 4) * 100;
```
Comment (janisozaur) [inline src/openrct2/peep/peep.c]: mixed indentation

File: src/openrct2/peep/peep.c [hunk 2]
```diff
@@ -10577,76 +10577,73 @@ static void peep_update_favourite_ride(rct_peep *peep, rct_ride *ride)
 
 /* rct2: 0x00695555 */
 static sint16 peep_calculate_ride_value_satisfaction(rct_peep* peep, rct_ride* ride) {
```
Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: Brace should go on its own line.

File: src/openrct2/peep/peep.c [hunk 3]
```diff
@@ -9568,7 +9568,7 @@ int peep_pathfind_choose_direction(sint16 x, sint16 y, uint8 z, rct_peep *peep)
 		 * the last 4 thin junctions that the peep walked through.
 		 * For each of these 4 thin junctions the peep remembers
 		 * those edges it has not yet taken.
-                 * If a peep returns to one of the 4 thin junctions that it
+				 * If a peep returns to one of the 4 thin junctions that it
```
Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: Odd alignment. (There are some other cases.)

File: src/openrct2/peep/peep.c [hunk 4]
```diff
@@ -10577,76 +10577,73 @@ static void peep_update_favourite_ride(rct_peep *peep, rct_ride *ride)
 
 /* rct2: 0x00695555 */
 static sint16 peep_calculate_ride_value_satisfaction(rct_peep* peep, rct_ride* ride) {
-    if (gParkFlags & PARK_FLAGS_NO_MONEY){
-        return -30;
-    }
+	if (gParkFlags & PARK_FLAGS_NO_MONEY){
```
Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: Same stuff with the brace, I think?

File: src/openrct2/peep/peep.c (hunk 2 above)
Comment (marijnvdwerf) [inline src/openrct2/peep/peep.c]: Why?

Comment (Gymnasiast) [inline src/openrct2/peep/peep.c]: Because the coding style says so.

**Comment by IntelOrca** [comment]
Look this isn't a PR to reformat peep.c, I think I will retract the spaces to tab change.

File: src/openrct2/peep/peep.c (hunk 2 above)
Comment (marijnvdwerf) [inline src/openrct2/peep/peep.c]: Look at the filename. The style guide is about C++.

**Pull request merged**